src/Microsoft.ML.TorchSharp/Utils/Range.cs

```diff
@@ -134,5 +137,293 @@ public override string ToString()
             var stop = !range.End.IsFromEnd ? new long?(range.End.Value) : range.End.Value == 0 ? null : new long?(-1 * range.End.Value);
             return TensorIndex.Slice(start, stop);
         }
+
+        /// <summary>
+        /// Slices the specified array using the specified range.
+        /// </summary>
+        public static T[] GetSubArray<T>(T[] array, Range range)
```

Comment:
Was this API on Range before?  I don't see it there -- looks like this was on RuntimeHelpers.  We should preserve that factoring.